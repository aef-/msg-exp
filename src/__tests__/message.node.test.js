/**
 * @jest-environment node
 */

import './message.common';
