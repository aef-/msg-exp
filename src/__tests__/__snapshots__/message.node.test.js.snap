// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message Component should correctly parse message with HTML tags and no variables 1`] = `
<span>
  This is a message 
  <a
    href="test"
  >
    <strong
      id="test-id"
    >
      test
    </strong>
  </a>
  !
</span>
`;

exports[`Message Component should return React tree when HTML is passed 1`] = `
<span>
  This is a 
  <strong
    data-attr="1"
    id="strong-id"
  >
    a very 
    STRONG!!
  </strong>
   message.
</span>
`;
